packages/pyright-internal/typeshed-fallback/stdlib/typing.pyi

```diff
@@ -135,7 +135,11 @@ AsyncContextManager = AbstractAsyncContextManager
 # This itself is only available during type checking
 def type_check_only(func_or_cls: _F) -> _F: ...
 
-Any = object()
+if sys.version_info < (3, 11):
+    Any: _SpecialForm
+else:
+    class _AnyMeta(type): ...
+    class Any(metaclass=_AnyMeta): ...
```

Comment:
not a good solution but i think the issue is that pyright uses different logic on types in `typing.pyi` based on whether it's an assignment or not
```suggestion
    class _Any(metaclass=_AnyMeta): ...
    Any = _Any
```